Gems/GraphCanvas/Code/Source/Components/NodePropertyDisplays/VectorNodePropertyDisplay.cpp

```diff
@@ -302,13 +358,27 @@ namespace GraphCanvas
         {
             m_proxyWidget = new QGraphicsProxyWidget();
 
+            QWidget* widgetContainer  = new QWidget(nullptr);
+            QHBoxLayout* layout = new QHBoxLayout(widgetContainer);
+            widgetContainer->setSizePolicy(QSizePolicy::MinimumExpanding, QSizePolicy::Fixed);
+            
+            layout->setAlignment(Qt::AlignLeft);
+            layout->setContentsMargins(0, 0, 0, 0);
+
+            m_button = new QToolButton();
+            m_button->setSizePolicy(QSizePolicy::Fixed, QSizePolicy::Fixed);
+            m_button->setVisible(false);
+            QObject::connect(m_button, &QToolButton::clicked, [this] () {
+                m_dataInterface->OnPressButton();
+            });
```

Comment:
Instead of having the submit value event it might be simpler to just call UpdateDisplay after m_dataInterface->OnPressButton(). If you only wanted to do it if the value changed then OnPressButton could return a bool.